examples/launch_height_sagemaker.py

```diff
@@ -56,14 +56,10 @@
 
     trial_backend = SageMakerBackend(
         # we tune a PyTorch Framework from Sagemaker
-        sm_estimator=PyTorch(
+        sm_estimator=basic_cpu_instance_sagemaker_estimator(
```

Comment:
Regarding this one in all change bellow in examples, I believe the following would be better:

```python
from syne_tune.backend.sagemaker_backend.estimators import  pytorch_latest_framework, pytorch_latest_py_version
...
sm_estimator=PyTorch(
  entry_point=str(entry_point),
  instance_type="ml.m5.large",
  instance_count=1,
  framework_version=pytorch_latest_framework,
  py_version=pytorch_latest_py_version,
  ...
)
```

the reason is that otherwise, the examples hides a bunch of things to a user of sagemaker (he may have no idea that pytorch estimator is used or which instance type is used whereas it is quite critical to get this)

Reply:
> BTW: From these issues and PR, I sense that the underlying problem here is that since SageMaker framework containers are not always up-2-date, we run into issues that certain things are available as dependencies (including AWS features itself) to us locally, but not when we do remote launching.

SageMaker frameworks are updated quite frequently, however one gets the version specified (the version of frameworks were for instance update to move away from python 3.6). As long as the examples runs, I don't think it is a problem.

Regarding building our containers, we can do it but it adds operational pain which is why we removed this as the default option.

Reply:
OK makes sense, changed.

Reply:
Nit: Either capitalize these constants, or make them functions.

Reply:
I would vote for SM containers untill we run into a dependency issue we cannot fix by updating the container version (assuming 1-2 weeks leeway for SM containers to catch up). I would rather keep it simple untill we have no choice since adding any maintanance overhead slows down developent.